src/platform/egl/surface/android_surface.rs

```diff
@@ -379,7 +377,7 @@ impl Device {
     ///
     /// This will be `GL_TEXTURE_2D` or `GL_TEXTURE_RECTANGLE`, depending on platform.
     #[inline]
-    pub fn surface_gl_texture_target(&self) -> GLenum {
+    pub fn surface_gl_texture_target(&self) -> u32 {
```

Comment:
Gleam still has an `GLenum` type and it's defined as `c_uint`, so I'm curious why this is changing here? https://docs.rs/gleam/latest/gleam/gl/types/type.GLenum.html

Reply:
There is no gleam in surfman, but c_uint = u32 (most of the time).

Reply:
Oh, of course. Thanks.